src/prefect/tasks/dbt/dbt.py

```diff
@@ -70,7 +70,7 @@ def __init__(
         overwrite_profiles: bool = False,
         profiles_dir: str = None,
         set_profiles_envar: bool = True,
-        dbt_kwargs: dict = None,
+        dbt_kwargs: dict = {},
```

Comment:
Mutable defaults in Python are [generally not a good idea](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments); if this kwarg should default to an empty dictionary, the place to do it would be in line 86 below:
```python
self.dbt_kwargs = dbt_kwargs or {}
```